packages/datepickers/src/elements/Datepicker/Datepicker.tsx

```diff
@@ -30,66 +30,42 @@ import { DatepickerContext } from './utils/useDatepickerContext';
 import { StyledMenu, StyledMenuWrapper } from '../../styled';
 
 export interface IDatepickerProps {
-  /**
-   * The selected date to display
-   */
+  /** Sets the selected date */
   value?: Date;
   /**
-   * Returns the parsed date
+   * Handles changes to the selected date
+   *
+   * @param {Date} date The selected date to use
    */
   onChange?: (date: Date) => void;
-  /**
-   * Allows customization of date formatting within input.
-   */
+  /** Formats the date */
   formatDate?: (date: Date) => string;
```

Comment:
This should be a function description with `@param` and `@returns` additions.

@austingreendev please fill-in details re: the functionality that this prop provides.

Reply:
This is the "output" string that is displayed in the input when a date is selected.

Reply:
Updated in latest push.